tests/specs/benchmarks/verification.k

```diff
@@ -176,81 +156,56 @@ module VERIFICATION
   // Arithmetic
   // ########################
 
-    rule (A +Int I2) +Int (I3 -Int A) => I2 +Int I3
-      [simplification]
-    rule (A +Int I2) -Int (A +Int I3) => I2 -Int I3
-      [simplification]
-    rule (A +Int I2) -Int (I3 +Int A) => I2 -Int I3
-      [simplification]
+    rule (A +Int I2) +Int (I3 -Int A) => I2 +Int I3 [simplification]
+    rule (A +Int I2) -Int (A +Int I3) => I2 -Int I3 [simplification]
+    rule (A +Int I2) -Int (I3 +Int A) => I2 -Int I3 [simplification]
 
-    rule (I1 +Int I2) -Int (I3 +Int I1) => I2 -Int I3
-      [simplification]
+    rule (I1 +Int I2) -Int (I3 +Int I1) => I2 -Int I3 [simplification]
 
-    rule A -Int (A -Int I1) => I1
-      [simplification]
+    rule A -Int (A -Int I1) => I1 [simplification]
 
-    rule (A +Int I1) -Int A => I1
-      [simplification]
+    rule (A +Int I1) -Int A => I1 [simplification]
 
-    rule (A +Int B) +Int (C -Int B) => A +Int C
-      [simplification]
+    rule (A +Int B) +Int (C -Int B) => A +Int C [simplification]
 
-    rule (A +Int B +Int C) -Int (A +Int D) => B +Int (C -Int D)
-      [simplification]
+    rule (A +Int B +Int C) -Int (A +Int D) => B +Int (C -Int D) [simplification]
 
-    rule (A +Int B +Int C +Int D +Int E) -Int (A +Int C +Int F) => B +Int D +Int E -Int F
-      [simplification]
+    rule (A +Int B +Int C +Int D +Int E) -Int (A +Int C +Int F) => B +Int D +Int E -Int F [simplification]
 
     // Simplify Cmem(_,_) - Cmem(_, _)
-    rule A +Int (I1 -Int A) => I1
-      [simplification]
-    rule (I1 +Int (A -Int I0)) +Int (I2 -Int A) => I1 +Int (I2 -Int I0)
-      [simplification]
+    rule A +Int (I1 -Int A) => I1 [simplification]
+    rule (I1 +Int (A -Int I0)) +Int (I2 -Int A) => I1 +Int (I2 -Int I0) [simplification]
 
     // safeMath mul check c / a == b where c == a * b
-    rule (X *Int Y) /Int X => Y  requires X =/=Int 0
-      [simplification]
-    rule chop(X *Int Y) /Int X ==K Y => false  requires X =/=Int 0 andBool pow256 <=Int X *Int Y
-      [simplification]
+    rule (X *Int Y) /Int X => Y  requires X =/=Int 0 [simplification]
+    rule chop(X *Int Y) /Int X ==K Y => false  requires X =/=Int 0 andBool pow256 <=Int X *Int Y [simplification]
```

Comment:
Is this lemma needed? It seems to actually be false to me. Take for example, if `X ==Int 1`, `Y ==Int pow256 +Int 1`. Then you get `chop(X *Int Y) => 1`, and `1 /Int 1 => 1` and `1 ==K 1 => false`.

This can be fixed by either saying `#rangeUInt(256, Y)`, or by just removing this lemma if it's not needed.